File: clients/dart/lib/model/access_token.dart
part of api;


@Entity()
class AccessToken {
  
  @Property(name: 'id')
  String id = null;
  
/* time to live in seconds (2 weeks by default) */
  @Property(name: 'ttl')
  double ttl = 1209600.0;
  

  @Property(name: 'created')
  DateTime created = null;
  

  @Property(name: 'userId')
  double userId = null;
  
  AccessToken();

  @override
  String toString()  {
    return 'AccessToken[id=$id, ttl=$ttl, created=$created, userId=$userId, ]';
  }

}

File: clients/typescript-angular/API/Client/AccessToken.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface AccessToken {

        id: string;

        /**
         * time to live in seconds (2 weeks by default)
         */
        ttl?: number;

        created?: date;

        userId?: number;
    }

}
